frontend/src/views/PollView.vue

```diff
@@ -197,16 +203,16 @@ onMounted(async () => {
             v-for="(choice, choiceId) in poll.ipfsParams.choices"
             :key="choiceId"
             :class="{
-              selected: selectedChoice === choiceId || choiceId === winningChoice,
+              selected: Number(selectedChoice) === choiceId || choiceId === Number(winningChoice),
```

Comment:
Number?

Reply:
Yea, the contract's returning bigints the JS file contains Numbers, one needs to be cast to another for comparison. There is probably a cleaner way.